components/perl/Encode-Locale/test/results-all.master

```diff
@@ -0,0 +1,15 @@
+t/alias.t ...... ok
+# ENCODING_LOCALE is ascii
+t/arg.t ........ ok
+t/env.t ........ ok
+t/tain.t ....... ok
+t/warn_once.t .. ok
+All tests successful.
+
+Test Summary Report
+-------------------
+t/arg.t      (Wstat: 0 Tests: 4 Failed: 0)
+  TODO passed:   1-2
+Files=5, Tests=28
+Result: PASS
+make[1]: Leaving directory '$(@D)'
```

Comment:
I have slightly different test resullts as I have LC_ALL=C.UTF-8:
t/alias.t ...... ok
# ENCODING_LOCALE is UTF-8
t/arg.t ........ ok
t/env.t ........ ok
t/tain.t ....... ok
t/warn_once.t .. ok
All tests successful.

Test Summary Report
-------------------
t/arg.t      (Wstat: 0 Tests: 4 Failed: 0)
  TODO passed:   1-4
Files=5, Tests=28
Result: PASS

Reply:
When I touch Encode::Locale next time, I'll try remember to specify `LANG` and `LC_ALL` in the test env in the `Makefile`, so that difference goes away.  Using `C.UTF-8` for the tests seems like a good idea to me.